src/plugins/interactive_setup/server/routes/enrollment/enroll.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import type { RouteDefinitionParams } from '../';
+
+/**
+ * Defines routes to deal with Elasticsearch `enroll_kibana` APIs.
+ */
+export function defineEnrollRoutes({ router }: RouteDefinitionParams) {
+  router.post(
+    {
+      path: '/internal/interactive_setup/enrollment/enroll',
```

Comment:
I don't think there will be multiple enrolment endpoints so `/internal/interactive_setup/enroll` should be enough

Reply:
Okay, will use `/internal/interactive_setup/enroll` for now.

Reply:
Done